shadows/framework/src/main/java/org/robolectric/shadows/ShadowCountDownTimer.java

```diff
@@ -1,60 +1,83 @@
 package org.robolectric.shadows;
 
+import static org.robolectric.util.reflector.Reflector.reflector;
+
 import android.os.CountDownTimer;
 import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
 import org.robolectric.annotation.RealObject;
 import org.robolectric.shadow.api.Shadow;
 import org.robolectric.util.ReflectionHelpers.ClassParameter;
+import org.robolectric.util.reflector.Accessor;
+import org.robolectric.util.reflector.Direct;
+import org.robolectric.util.reflector.ForType;
 
 @Implements(CountDownTimer.class)
 public class ShadowCountDownTimer {
   private boolean started;
-  private long countDownInterval;
-  private long millisInFuture;
-
-  @RealObject CountDownTimer countDownTimer;
+  @RealObject private CountDownTimer realCountDownTimer;
 
   @Implementation
   protected void __constructor__(long millisInFuture, long countDownInterval) {
-    this.countDownInterval = countDownInterval;
-    this.millisInFuture = millisInFuture;
     this.started = false;
     Shadow.invokeConstructor(
         CountDownTimer.class,
-        countDownTimer,
+        realCountDownTimer,
         ClassParameter.from(long.class, millisInFuture),
         ClassParameter.from(long.class, countDownInterval));
   }
 
   @Implementation
   protected synchronized CountDownTimer start() {
```

Comment:
Can we have a temporary system property to continue to avoid calling the real code? There are unfortunately a decent number of tests that will fail at Google if the real code is invoked, due to the dependency on the old behavior. This is probably true for other Robolectric tests using Gradle. A system property would give teams the ability to do a migration.

Something like:

```java
  /**
   * Previously, Robolectric shadowed the start method and made it a no-op. This allows tests to
   * use the old behavior during a migration.
   */
  static boolean useRealCountDownTimer() {
    return Boolean.parseBoolean(System.getProperty("robolectric.useRealCountDownTimer", "true"));
  }
```

And then 
```java
  @Implementation
  protected synchronized CountDownTimer start() {
    if (useRealCountDownTimer()) {
      return reflector(CountDownTimerReflector.class, realCountDownTimer).start();
    } else {
      return countDownTimer;
    }
```